data/clipper/NOTICE

```diff
@@ -28,7 +28,15 @@ You can obtain a copy of the raw data and its license at:
 
    https://511.org/developers/list/data/
 
-5. Station, stop and agency ID mappings for Clipper card are not provided by the aforementioned agencies. This information is available in Metrodroid's source repository, and is subject to the same license terms as Metrodroid.
+5. Golden Gate Transit (GGT) station names and locations are derived from GGT's GTFS feed, made available under the 511 Data Disseminator Agreement.
```

Comment:
Can this be merged with the (previous) SFBF notice? They both appear to be from the same website, and have the same terms and conditions.

Reply:
Done!